jupiter-rs/src/lru/cache.rs

```diff
@@ -563,6 +591,24 @@ mod tests {
                 .unwrap();
             assert_eq!(std::str::from_utf8(&result[..]).unwrap(), "$3\r\nbar\r\n");
 
+            // ...and ensure we see the key (without and with filtering...)
+            let result = dispatcher
+                .invoke(Request::example(vec!["LRU.KEYS", "test"]), None)
+                .await
+                .unwrap();
+            assert_eq!(
+                std::str::from_utf8(&result[..]).unwrap(),
+                "*1\r\n$3\r\nfoo\r\n"
+            );
+            let result = dispatcher
+                .invoke(Request::example(vec!["LRU.KEYS", "test", "fo"]), None)
+                .await
+                .unwrap();
+            assert_eq!(
+                std::str::from_utf8(&result[..]).unwrap(),
+                "*1\r\n$3\r\nfoo\r\n"
```

Comment:
Wouldnt it make more sense to put anotheer key in the test that doesn't match the filter, as currently this test would also be true if the filter would never be applied ?

Reply:
added another testcase